pkg/destroy/azure/azure.go

```diff
@@ -100,26 +101,85 @@ func New(logger logrus.FieldLogger, metadata *types.ClusterMetadata) (providers.
 }
 
 // Run is the entrypoint to start the uninstall process.
-func (o *ClusterUninstaller) Run() error {
+func (o *ClusterUninstaller) Run() (err error) {
+	var errs []error
+
 	o.configureClients()
 
-	o.Logger.Debug("deleting public records")
-	if err := deletePublicRecords(context.TODO(), o.zonesClient, o.recordsClient, o.privateZonesClient, o.privateRecordSetsClient, o.Logger, o.ResourceGroupName); err != nil {
+	group := o.InfraID + "-rg"
```

Comment:
this change seems like a rebase artifact after https://github.com/openshift/installer/pull/3920/files#diff-cfbf148955816bcfd4e1d89b1c863cc1

can you keep using the o.ResourceGroupName

Reply:
Fixed.